src/components/CardFlight.tsx

```diff
@@ -1,31 +1,31 @@
 import React, { Component } from 'react';
-import { View, Text, StyleSheet } from 'react-native';
+import { View } from 'react-native';
 import { DataTable, Divider } from 'react-native-paper';
 import { Ionicons } from '@expo/vector-icons';
-import Constants from 'expo-constants';
+import Styles from '../styles/MyFlights';
 
 export default class CardFlight extends Component {
   render() {
     return (
-      <View style={{ margin: 10 }}>
-        <View style={styles.container}>
-          <Text style={styles.text}>BEG</Text>
-          <Ionicons name='airplane-sharp' size={30} color='#6170f7' />
-          <Text style={styles.text}>Serbia</Text>
-        </View>
-
-        <View>
+      <View>
```

Comment:
The `<View>` component should be removed since it doesn't add any style

Reply:
Changes done